### ISSUE TITLE
Reviewer 1 Comment 5

### ISSUE THREAD

**Issue by ramess101** [ramess101/JCED_FOMMS_Manuscript#23]
@mrshirts @jpotoff @msoroush 

> On page 6 in the force fields section, authors mention that the bond stretching potential is neglected for all the force fields. They compared their result with those from Mick et al. but Mick also used fixed bond models. This is not valid justification. Have the authors conducted any tests showing that the bond stretching has no influence on the simulation result or is this reported from literatures in the similar systems? The same question comes to electrostatic interactions. Please show evidence that it is valid to ignore these interactions.

We compare with Mick et al because we are validating that our MBAR results agree with HR. There would actually be much less justification if we compared MBAR results that used fixed bonds with HR results that used flexible bonds, or vice versa. But I think the reviewer's point is really aimed at whether VLE calculations, in general, are sensitive to flexible or fixed bonds. I know there have been a few (older) studies that looked into the impact of bond treatment. I will try to find those. @jpotoff @msoroush can you think of any studies off the top of your head?

The question regarding electrostatic interactions is somewhat misplaced. One could argue that performing simulations with fixed or flexible bonds is a set-up choice (not strictly tied to the force field) whereas electrostatics are completely a model choice when fitting a force field. All UA or AUA models, that I am aware of, do not use point charges for normal, branched, and cyclic hydrocarbons. I don't think we need to vindicate this decision that has been made for the 
OPLS-UA, TraPPE, NERD, AUA4, TAMie, and MiPPE force fields. Any thoughts?

**Comment by jpotoff** [comment]
@ramess101 Well, both comments from the reviewer miss the mark.  We can't do a validation of a method by changing the models used in the calculations.  

For the alkynes, we did extra calculations to show that adding electrostatic interactions did not substantially improve the model for pure or mixture phase behavior (it's in the supporting info).  If it's not going to effect alkynes, is definitely not going to be an issue for alkanes (for VLE).  Could electrostatics and polarizability be important for the free energy of solvation of alkanes in water?  Maybe, but that's not the model, or the properties of interest here.

**Comment by ramess101** [comment]
@jpotoff 

> We can't do a validation of a method by changing the models used in the calculations.

Very well said. I will try to emphasize this point in my response. I will also mention your experience with alkynes.

**Comment by mrshirts** [comment]
Right, the point is not to determine whether bond stretching affects the results.  The goal is to show that the approach allows you to determine GCMC-determined properties at a large number of parameters very quickly.

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

Sorry for the delay. Here is my response for this reviewer comment. Please let me know if you have any objections, suggestions, or comments. 

In particular, @jpotoff and @msoroush do you object to me mentioning that flexible bonds were not yet available in GOMC? Although the real reason we used fixed bonds for NERD was to compare our results with Mick et al., perhaps the underlying reason Mick et al. utilized fixed bonds was because flexible bonds were not supported by GOMC (?). Or do you know of a study that definitively demonstrates that flexible bonds do not impact phase equilibria?

![image](https://user-images.githubusercontent.com/23408516/53688503-3fbb9e00-3d02-11e9-80ee-226db5089c58.png)

**Comment by jpotoff** [comment]
@ramess101 it seems to me that it would be more straightforward just to say the models used fixed bonds and remove any discussion of flexible bonds and their potential effect on the predicted phase behavior.  This is a paper on a method, so we use the models as originally developed to validate the method.

**Comment by ramess101** [comment]
@jpotoff So I am fine removing the discussion about agreement between fixed
and flexible. But the NERD model actually was developed with flexible
bonds, so we can’t exactly claim that we are using fixed bonds for the
purpose of complying with the model specifications. But we can reiterate
that we are verifying the method and our comparison data used fixed bonds.

**Comment by jpotoff** [comment]
@ramess101 We have tons of data for NERD from the branched alkanes paper, so if we need to address the fixed vs. flexible thing, we can provide a comparison between our results and the original NERD papers to show they are the same.

**Comment by ramess101** [comment]
@jpotoff Thats a good idea. I already have those data as well. I will make
some plots comparing the results and include it in our response and maybe
the SI.

**Comment by ramess101** [comment]
@jpotoff 

Haven't gotten around to the comparison plots yet. But here is how I modified the paragraph. I basically just removed the sentence about agreement between fixed and flexible. I am considering add a line at the end saying, "We verify that fixed and flexible bonds yield indistinguishable results in supporting information."

![image](https://user-images.githubusercontent.com/23408516/53782505-c91dcc80-3eca-11e9-8c03-f203b6614842.png)
![image](https://user-images.githubusercontent.com/23408516/53782550-06825a00-3ecb-11e9-922a-ce321108a951.png)

**Comment by ramess101** [comment]
@jpotoff @msoroush 

I have plotted the Mick et al. and Nath et al. values for the NERD potential for branched alkanes. Nath et al. only reported orthobaric densities, so that is all we can compare. The agreement is within the uncertainties for liquid density at all state points (although 2-methylbutane and 2,4-dimethylhexane appear to have a bias in rhol). The rhov agreement is not as good, but fairly reasonable. I recall when implementing the NERD potential, Mohammad pointed out that the parameter types are somewhat ambiguous. So maybe Mick et al. didn't use the exact same parameters for some of these compounds. Either way, I think this should be good enough to show that the phase equilibria are pretty similar.

![image](https://user-images.githubusercontent.com/23408516/53824550-6a3f6e00-3f31-11e9-811e-7ab3cb6b44bc.png)

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

Here is my response:

![image](https://user-images.githubusercontent.com/23408516/53829444-67e31100-3f3d-11e9-877f-03bb9c41d4fc.png)
![image](https://user-images.githubusercontent.com/23408516/53829459-70d3e280-3f3d-11e9-8041-6f6b3fad6f87.png)

**Comment by msoroush** [comment]
@ramess101 You might wanna avoid reporting data for the compounds that we were not sure which CH3 parameter to use.

**Comment by ramess101** [comment]
@msoroush 

Yeah, that's true. I was just trying to maximize the amount of data I could plot here.

This is all that we are sure about in the NERD manuscript:

![image](https://user-images.githubusercontent.com/23408516/53831380-f3f73780-3f41-11e9-82a4-9247b15672b0.png)

**Comment by ramess101** [comment]
@msoroush 

So we know isobutane (2-methylpropane) and 2-methylbutane are correct. 2-methylbutane was one of the bigger discrepancies. 2,4-dimethylhexane is the other concern, and it could be slightly more ambiguous to parameterize.
